dev-python/pythonfinder/files/2.0.5-fix-import-cached-property.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/pythonfinder/models/path.py b/src/pythonfinder/models/path.py
```

Comment:
DId you send this upstream somewhere?

Reply:
Yes, it is upstreamed. https://github.com/sarugaku/pythonfinder/pull/148